src/Formatter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types = 1);
+
+namespace LanguageServer;
+
+use LanguageServer\Protocol\{TextEdit, Range, Position, ErrorCode};
+use AdvancedJsonRpc\ResponseError;
+
+class Formatter
+{
+
+    /**
+     *
+     * @param string $content            
+     * @param string $uri            
+     *
+     * @return \LanguageServer\Protocol\TextEdit[]
+     * @throws \AdvancedJsonRpc\ResponseError
+     */
+    public function format(string $content, string $uri)
+    {
+        $path = uriToPath($uri);
+        $cs = new \PHP_CodeSniffer();
+        $cs->initStandard($this->findConfiguration($path));
```

Comment:
What I meant was that we need to handle URIs that don't use the `file://` scheme, but something like `vfs://`.